components/properties/src/exemplar_chars.rs

```diff
@@ -231,6 +216,5 @@ make_exemplar_chars_unicode_set_property!(
     /// assert!(!exemplars_index.contains("ng"));
     /// assert!(exemplars_index.contains("A"));
     /// ```
-
-    pub fn load_exemplars_index();
+    pub fn exemplars_index();
```

Comment:
Praise: Given where we're at I like the new pattern you came up with of `load_` implying fallibility